### ISSUE TITLE
A11y_Omni channel supervisor dashboard_Assign - Agent_Screen Reader: NVDA is not reading the number of search results found

### ISSUE THREAD

**Issue by InduPriya1805** [nvaccess/nvda#10167]
**Environment Details:**
URL: https://crm643345.crm.dynamics.com/​
User Name: admin@crm643345.OnMicrosoft.com​
Server version: 9.1.0000.7625​
Client version: 1.3.2119-190813-224452​

**Browser Details:**  Google Chrome 76.0.3809.132​
**OS Version:** Microsoft Windows Version 1903 (OS Build 18362.113)​

**Repro Steps:​**
1)Turn on NVDA
2)Hit the URL "https://crm643345.crm.dynamics.com/" and login with appropriate credentials​​
3)Tab till "Omni Channel for Customer Service" card and press enter​
4)Select any of the conversation in the grid which is in open state & Tab till "Assign" button and press enter​
5)Tab till search field and search anything and verify NVDA is reading number of search results found or not
​
**Actual Result:​**
NVDA is not reading the number of search results found

**Expected Result:​**
NVDA should read the status message number of search results found
​
**User Impact:​**
If NVDA does not read the message then screen reader users will not get to know how many results or found from their search

**MAS Reference:​**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment:**
[Bug_10167_NVDA not reading status message.pptx](https://github.com/nvaccess/nvda/files/3583366/Bug_10167_NVDA.not.reading.status.message.pptx)

**Comment by Adriani90** [comment]
cc: @josephsl

**Comment by michaelDCurran** [comment]
Closing on request from reporter privately by email.
